dbschema/producible.esdl

```diff
@@ -0,0 +1,12 @@
+module default {
+  abstract type Producible extending Resource {
+    scriptureReferences: multirange<int32>;
+  }
+}
+
+module Producible {
+  function sortScriptureReferences(scriptureReferences: multirange<int32>) -> multirange<int32> 
```

Comment:
Doesn't this happen automatically on `set`/save?

Reply:
Doesn't the sorting/minimizing happen when the multirange is created? Aka there's no such thing as an unsorted multirange?

Reply:
Hmm...it does.  So we have no need of a function at all then?